### ISSUE TITLE
MapStyle.Road is not defined

### ISSUE THREAD

**Issue by Alexbits** [arnaudleclerc/AzureMapsControl.Components#42]
Hi there,

It would be nice to have a MapStyle.Road ("road"). Without it, I see no way to add it to the StyleControlOptions. Am I missing something?

The code below will not work.

```
await _map.AddControlsAsync (
  new AzureMapsControl.Components.Controls.StyleControl(
                    new AzureMapsControl.Components.Controls.StyleControlOptions
                    {
                        Style = AzureMapsControl.Components.Controls.ControlStyle.Auto,
                        MapStyles = new[] { MapStyle.Road, MapStyle.SatelliteRoadLabels }
                    }, AzureMapsControl.Components.Controls.ControlPosition.BottomRight));
```

**Comment by arnaudleclerc** [comment]
Hi! You're right, I totally missed the Road style! I'll add it tomorrow morning and publish a new fix version with it 😊

**Comment by Alexbits** [comment]
Great. Thanks!

**Comment by arnaudleclerc** [comment]
I have just published a version `1.5.2` where you can now find a `MapStyle.Road`. 

```
@page "/"

@using AzureMapsControl.Components.Map
<AzureMap Id="map"
          CameraOptions="new CameraOptions { Type = AzureMapsControl.Components.Map.CameraType.Fly, Center= new AzureMapsControl.Components.Atlas.Position(11.581990, 48.143534), Zoom= 8.5, MinZoom = 3, MaxZoom = 12.5 }"
          StyleOptions="StyleOptions"/>

@code {
    public StyleOptions StyleOptions = new StyleOptions
    {
        Style = MapStyle.Road,
        ShowLogo = false,
        ShowFeedbackLink = false
    };
}
```

Check it out and feel free to reopen this issue if necessary!
